packages/debug/src/browser/view/debug-threads-widget.ts

```diff
@@ -128,6 +130,19 @@ export class DebugThreadsWidget extends VirtualWidget {
         this.threads = [];
     }
 
+    protected onStoppedEvent(event: DebugProtocol.StoppedEvent): void {
+        const body = event.body;
+
+        if (body.threadId) {
+            switch (body.reason) {
+                case 'start': {
```

Comment:
Should we update threads independently of the `reason`?

Reply:
I wanted to be specific to avoid spamming the thread update, but I'm happy to update on any stop event